modules/text2vec-openai/module.go

```diff
@@ -31,7 +31,8 @@ import (
 )
 
 const (
-	Name = "text2vec-openai"
+	Name           = "text2vec-" + ModuleProvider
```

Comment:
Does this work if you set `Name` with `ModuleProvider`, before you set the `ModuleProvider` value in the next line? :o